bindata/cloud-network-config-controller/002-rbac.yaml

```diff
@@ -39,6 +47,7 @@ rules:
   - get
   - create
   - update
+  - patch
```

Comment:
Why was patch added? It's not "tightening up" and is not needed (presumably since it has worked without it)